.yamato/mobile-build-and-test.yml

```diff
@@ -0,0 +1,90 @@
+2021.2_Build_Android_player:
+  name: 2021.2 Build Android player
+  agent:
+      type: Unity::VM
+      # Any generic image can be used, no need to have Android tools in the image for building
+      # All Android tools will be downloaded by unity-downloader-cli
+      image: package-ci/win10:stable
+      flavor: b1.small
+
+  commands:
+    - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - mkdir C:\TMP
```

Comment:
curious why we care where the editor is ? the default location is always going to be .editor in the current directory which is where you launch the tests from anyway. I generally prefer to just use the default location.

Reply:
The issue is that the path to the android tools become greater than the maximum path length on Windows and commands will fail when trying to use those long paths. Probably a better solution would be to enable "longPaths" for Windows rather than moving the location of the Editor because it is very very close to the limit as it is. I can try that and see what happens.

Reply:
If that is the case then why not build it on macos? It supports building android and doesn't have the long path issue ?